src/app/_components/RolesList/RolesListTajemniczy.tsx

```diff
@@ -0,0 +1,75 @@
+'use client';
+
+import {useMemo} from "react";
+import {RolesTajemniczy} from "@/roles";
+import {ModifiersTajemniczy} from "@/modifiers";
+import {RoleOrModifierTypes} from "@/constants/rolesAndModifiers";
+import {Teams} from "@/constants/teams";
+import {RolesListContext, TeamFilters, TypeFilters, useFilters, useSearch} from "./hooks";
+import {Search} from "./Search";
+import {Filters} from "./Filters";
+import {RoleCard} from "./RoleCard/RoleCard";
+
+const RolesAndModifiers = [...RolesTajemniczy, ...ModifiersTajemniczy];
+
+export const RolesListTajemniczy = () => {
```

Comment:
Ten komponent i `RolesList` to te same komponenty, więc nie ma sensu kopiować całego kodu. Raczej powinniśmy mieć wspólny komponent, który potrafi wyświetlić listę, którą ktoś mu poda. W jednym miejscu powinniśmy mieć listę dla Tajemniczego, w jednym dla nas.